### DISCUSSION TITLE
Call back response "message": "Internal server error" from Nova2

### DISCUSSION THREAD

**Discussion by ShaiStudny** [deepgram/community#876] [General help] [answered]
When working with the Nova2 model, I got a few responses with 
"statusCode": 500,
    "message": "Internal server error",
When I look at the logs in the account UI the failed requests are not shown in the log (also tried looking up by the request ID and cannot find them).
Most requests come back fine, I have 3-4 of these 500 code.

Any thoughts on why this happens and how to troubleshoot?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Reply by ShaiStudny** [reply]
I sent this video https://geoedge-ctv.s3.amazonaws.com/automation/Media_files/Audio_files/mpeg_audio_politics.mpeg
to the regular /v1/listen?model=nova-2-video&detect_language=true&detect_topics=true
we are using js sdk version 3.5
I got the request ID b017c5a4-320e-4545-a106-90e80ea2fef6
Jul 30, 2024, 13:10:55.866 (UTC+03:00)
I do not see it in the log under deepgarm site

**Reply by jpvajda** [reply, marked as answer]
@ShaiStudny  I actually can't find that request ID in our logs, which means It may have never been processed by Deepgram. That can happen from time to time, especially if we were experience a service issue.

Could you try again and let me know if you see a similar problem?


As a note: i did download your audio file and try to run it with the same API params I get an error:

``` json
{
    "err_code": "PAYLOAD_ERROR",
    "err_msg": "Failed to decode payload as UTF-8.",
    "request_id": "b46b8fed-4d32-4848-8bf2-1f3bddb16409"
}
```


So you might want to check the encoding on this file  and change if necessary to UTF-8. You can use ffmpeg to fix the the encoding of the file.
